syncer/syncer.go

```diff
@@ -1352,6 +1364,14 @@ func (s *Syncer) syncDML(
 		if len(jobs) == 0 {
 			return 0, nil
 		}
+		// set this batch of job TS when this batch is started.
+		s.updateReplicationJobTS(jobs[0], workerJobIdx)
```

Comment:
How about move this to L1436, we can update the ts only when we receive the first job, this should be more accurate and reasonable.
```
for {
  ....
  case sqlJob, ok := <-jobChan:
     if len(jobs) == 0 {
      s.updateReplicationJobTS(sqlJob, workerJobIdx)
    }
}
```

Reply:
fixed in https://github.com/pingcap/dm/pull/2060/commits/6b1bfb9bc0cfa1d0cb06ea150f8f5b3dcd2717bb